contrib/net/http/option.go

```diff
@@ -197,3 +199,9 @@ func RTWithAnalyticsRate(rate float64) RoundTripperOption {
 		}
 	}
 }
+
```

Comment:
Can you add a comment here for this new public function? Perhaps:
`// RTWithPropagation enables/disables propagation for tracing headers. Disabling propagation will disconnect this trace from any downstream traces`